doc/question.md

```diff
@@ -45,22 +47,24 @@ The `info.json` file for each question defines properties of the question. For e
     "uuid": "cef0cbf3-6458-4f13-a418-ee4d7e7505dd",
     "title": "Addition of vectors in Cartesian coordinates",
     "topic": "Vectors",
-    "tags": ["Cartesian", "graphical"],
+    "tags": ["secret", "graphical"],
     "type": "v3"
 }
 ```
 
-- `title` gives a student-visible title for the question.
-- `topic` is the part of the course that this question belongs to (like the chapter in a textbook).
-- `tags` (optional) stores any other aspects of the questions, for sorting and searching (these can be anything).
-- `clientFiles` (optional) lists the files that the client (student's webbrowser) can access. Anything in here should be considered viewable by the student.
-- `type` specifies the question format and should be `"v3"` for the current PrairieLearn question format.
-
-## Partial credit
-
-By default all v3 questions award partial credit. For example, if there are two numeric answers in a question and only one of them is correct then the student will be awarded 50% of the available points.
+* [Format specification for question `info.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoQuestion.json)
 
-To disable partial credit for a question, set `"partialCredit": false` in the `info.json` file for the question. This will mean that the question will either give 0% or 100%, and it will only give 100% if every element on the page is fully correct.
+Property | Type | Description
+--- | --- | ---
+`uuid` | string | Unique identifier (UUID v4). E.g., `"8b4891d6-64d1-4e89-b72d-ad2133f25b2f"`. These can be obtained from https://www.uuidgenerator.net (Required: no default)
```

Comment:
Nit: would prefer semicolons here; for cases like `Optional; default: none`, I think that's a little clearer than `Optional: default: none`.

```suggestion
`uuid` | string | Unique identifier (UUID v4). E.g., `"8b4891d6-64d1-4e89-b72d-ad2133f25b2f"`. These can be obtained from https://www.uuidgenerator.net (Required; no default)
```

Reply:
good catch, this was just an error 😄